components/active/quiz/Results.vue

```diff
@@ -0,0 +1,183 @@
+<template>
+  <h1 class="my-12 text-center text-4xl">
+    <AtomsTextFirstColoredWord
+      :text="t('page.quiz.result.header', { test: response.quiz.title })"
+    />
+  </h1>
+
+  <div class="mb-4 flex w-full justify-start overflow-y-auto md:justify-center">
+    <div class="w-full min-w-[768px] md:w-3/4">
+      <ActiveQuizDiploma :response="response" />
+    </div>
+  </div>
+
+  <div class="container mx-auto flex flex-col items-center justify-center">
+    <client-only v-if="response.isSubmitter">
```

Comment:
`isSubmitted` ?

Reply:
This variable indicates if the current user is the one who submitted this quiz response, thus `isSubmitter`.

Reply:
Gotcha!